lib/jnpr/junos/device.py

```diff
@@ -52,6 +52,10 @@
 
 logger = logging.getLogger("jnpr.junos.device")
 
+# Setup paramiko logging
```

Comment:
You can get depend library logs using the snippet below. Adding separate logging files for third-party libraries is not acceptable.
```
import logging
 logging.basicConfig(level=logging.DEBUG)
```